cmd/kueueviz/frontend/src/useWebSocket.js

```diff
@@ -38,7 +38,13 @@ const useWebSocket = (url) => {
 
     ws.onerror = (err) => {
       console.error("WebSocket error:", err);
-      setError("WebSocket connection error");
+      const errorMessage = `
+      Failed to fetch data from WebSocket: ${err.message}.
+      Backend URL: ${fullUrl}
+      REACT_APP_WEBSOCKET_URL: ${env.REACT_APP_WEBSOCKET_URL}
```

Comment:
fullUrl variables should include REACT_APP_WEBSOCKET_URL. Not sure that we need to duplicate it.